BlazorMarkDownAppJwt/BlazorMarkDownAppJwt.Server.UnitTests/Server/Controllers/AuthControllerTest.cs

```diff
@@ -0,0 +1,193 @@
+﻿using BlazorMarkDownAppJwt.Server.Controllers;
+using BlazorMarkDownAppJwt.Server.Entities;
+using BlazorMarkDownAppJwt.Server.Services.Users;
+using BlazorMarkDownAppJwt.Shared;
+using Moq;
+
+namespace BlazorMarkDownAppJwt.UnitTests.Server.Controllers
+{
+    public class AuthControllerTest
+    {
+        private Mock<IUserService> UserServiceMoq { get; set; }
+
+        private AuthController Controller { get; set; }
+
+        private CancellationToken CancellationToken { get; set; }

```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.threading.cancellationtoken?view=net-6.0

if you need cancellation token you should take it from CancellationTokenSource